CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
```

Comment:
This doesn't seem fully inclusive.  Why not instead say something like `harassment-free experience for everyone.` (full stop) as then that is fully inclusive instead of some arbitrary traits that have nothing to do with ones ability to program?